app/ui/faviconProgressbar.js

```diff
@@ -0,0 +1,48 @@
+const { platform } = require('../utils');
+const assets = require('../../common/assets');
+
+module.exports.updateFavicon = function(percentageString) {
+  if (platform() === 'web') {
+    const percentage = parseInt(percentageString.replace('%', ''));
+    if (percentage === 0 || percentage === 100) {
+      const link = document.querySelector("link[rel*='icon']");
+      link.type = 'image/png';
+      link.href = assets.get('favicon-32x32.png');
+      document.getElementsByTagName('head')[0].appendChild(link);
+      return;
+    }
+    const link = document.querySelector("link[rel*='icon']");
+    const canvas = document.createElement('canvas');
+    const size = 32;
+
+    const lineWidth = 5;
```

Comment:
I don't like that you're using lineWidth here and in the function as a parmeter. This variable name should be more specific...consider `loaderWidth`